pkg/minikube/notify/notify.go

```diff
@@ -99,16 +100,23 @@ func getJson(url string, target *releases) error {
 }
 
 func getLatestVersionFromURL(url string) (semver.Version, error) {
```

Comment:
nit: I noticed that this method will prompt you to download v0.9.0 even if you are running v0.10.0 if we haven't "released" v0.10.0 in our releases.json.  Should we do a greater than check in here?